estate/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, fields
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "Create real estate properties and keep track of status"
+
+    name = fields.Char(required=True)
```

Comment:
Comment that also applies to all other fields: also add the string parameter so that the field can get translated to other languages :+1: